src/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.cpp

```diff
@@ -146,6 +151,10 @@ Unique<Earth::Impl> Earth::ImplFromType(const Earth::Type& aType, const Director
     {
         return std::make_unique<ExponentialImpl>(aType);
     }
+    else if (aType == Earth::Type::Undefined)
```

Comment:
Move this `if` statement first to follow enum declaration ordering (simpler to maintain).